docs/sql-data-sources-text.md

```diff
@@ -38,3 +36,35 @@ Spark SQL provides `spark.read().text("file_name")` to read a file or directory
 </div>
 
 </div>
+
+## Data Source Option
+
+Data source options of text can be set via:
+* the `.option`/`.options` methods of
+  *  `DataFrameReader`
+  *  `DataFrameWriter`
+  *  `DataStreamReader`
+  *  `DataStreamWriter`
```

Comment:
Can you also add CREATE TABLE with OPTIONS clause? (see https://github.com/apache/spark/pull/32546#discussion_r638688399). We will fix other pages too in another Pr.

Reply:
Oh, got it. I'll create the new JIRA and address the other pages, too.

Reply:
Just created JIRA here: SPARK-35528